src/err/err_state.rs

```diff
@@ -118,6 +118,12 @@ impl PyErrState {
             ptraceback: unsafe {
                 Py::from_owned_ptr_or_opt(
                     pvalue.py(),
+                    #[cfg(GraalPy)]
+                    ffi::PyObject_GetAttrString(
```

Comment:
@timfel for difference like this I guess one day we GraalPy will implement these APIs. What's the best way for us to track these and have a plan to resolve these once GraalPy adds support?

Reply:
I think the best option is that I open issues like https://github.com/oracle/graalpython/issues/392 and we make sure on our side to fix these workarounds on the PyO3 side as soon as we fix those issues. What do you think?

Reply:
Sure thing, that seems good to me, thanks 👍